### PR TITLE
Fix some TODOs

### PR DESCRIPTION
**Pull request by DaughterOfMars**
# Description of change

Fix some TODOs or remove comments which are no longer needed.

### CODE REVIEW

File: sdk/examples/client/block/02_block_custom_parents.rs [hunk 1]
```diff
@@ -39,13 +43,32 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
...
+                    .await?
+                    .reference_mana_cost,
+            },
+        )
+        .finish_block_body()?,
+    )
+    .sign_ed25519(&secret_manager, Bip44::new(IOTA_COIN_TYPE))
+    .await?;
 
     println!("{block:#?}");
```
Comment (Thoralf-M) [inline sdk/examples/client/block/02_block_custom_parents.rs:74]: Add a `client.post_block(&block).await?;` or remove the println!() in L75 and update the comments?

File: sdk/src/client/api/block_builder/mod.rs [hunk 2]
```diff
@@ -24,15 +24,10 @@ impl Client {
...
         let issuing_time = {
-            #[cfg(feature = "std")]
             let issuing_time = std::time::SystemTime::now()
                 .duration_since(std::time::UNIX_EPOCH)
                 .expect("Time went backwards")
                 .as_nanos() as u64;
-            // TODO no_std way to have a nanosecond timestamp
-            // https://github.com/iotaledger/iota-sdk/issues/647
-            #[cfg(not(feature = "std"))]
-            let issuing_time = 0;
```
Comment (Thoralf-M) [inline sdk/src/client/api/block_builder/mod.rs]: This is also an issue for wasm? https://github.com/rust-lang/rust/issues/48564

Comment (DaughterOfMars) [reply sdk/src/client/api/block_builder/mod.rs]: used instant instead

File: sdk/examples/client/block/02_block_custom_parents.rs (hunk 1 above)
Comment (DaughterOfMars) [reply sdk/examples/client/block/02_block_custom_parents.rs:74]: done

**Review by Thoralf-M — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

**Pull request merged**